hassio/src/addon-view/hassio-addon-audio.ts

```diff
@@ -123,14 +123,14 @@ class HassioAddonAudio extends LitElement {
   }
 
   private _setInputDevice(ev): void {
-    const device = ev.detail.device;
+    const device = ev.detail.item.attributes.device.value;
```

Comment:
set device as property and use `ev.detail.item.device` or use `ev.detail.item.getAttribute("device")`

Reply:
Property looked cleaner.

Reply:
Yeah but now your `attr-for-selected` is missing

Reply:
Right...